### ISSUE TITLE
Hello, thanks for your sharing!

### ISSUE THREAD

**Issue by Kokakuu** [xxlya/BrainGNN_Pytorch#7]
I'm trying to reproduce the reported results in your paper BrainGNN, and I run into some problems. In the train and val process, I've found that the model is not trainable at all for the train loss droped while the val loss and acc even got worse. For example:
      Epoch: 236, Train Loss: 0.9192825, Train Acc: 0.6586151, Test Loss: 1.0345690, Test Acc: 0.5217391
      train...........
      LR 4.8828125e-06
      testing...........
      *====**
      0m 2s
      Epoch: 237, Train Loss: 0.9245522, Train Acc: 0.6618357, Test Loss: 1.0337465, Test Acc: 0.5265700
      train...........
      LR 4.8828125e-06
      testing...........
      *====**
      0m 2s
      Epoch: 238, Train Loss: 0.9274187, Train Acc: 0.6602254, Test Loss: 1.0354698, Test Acc: 0.5169082
      train...........
      LR 2.44140625e-06
      testing...........
      *====**
      0m 2s
      Epoch: 239, Train Loss: 0.9264973, Train Acc: 0.6602254, Test Loss: 1.0384360, Test Acc: 0.5265700
      train...........
      LR 2.44140625e-06
      testing...........
      *====**
      0m 2s
      Epoch: 240, Train Loss: 0.9233875, Train Acc: 0.6602254, Test Loss: 1.0387096, Test Acc: 0.5217391
      train...........
      LR 2.44140625e-06
      testing...........
      *====**
      0m 2s
      Epoch: 241, Train Loss: 0.9042460, Train Acc: 0.6634461, Test Loss: 1.0334812, Test Acc: 0.5217391

The only code I changed is the train_dataset. I just use the index to develop the three datasets but not by masks. 
      # train_dataset = dataset[train_mask]
      # val_dataset = dataset[val_mask]
      # test_dataset = dataset[test_mask]
      train_dataset = dataset[tr_index]
      val_dataset = dataset[val_index]
      test_dataset = dataset[te_index] 

I've also obstructed by this problem in several baseline graph model such as GAT and GCN. 
Looking forward to your reply.

**Comment by xxlya** [comment]
Hi, our hyper-parameters are not optimized on ABIDE dataset. I am trying to share the data used in our paper but it takes some process as I am not the data owner. If you have the HCP dataset at hand, please try on it.

**Comment by Kokakuu** [comment]
Hello again!
Thanks for your reply and please excuse me for my abrupt questions for I just recently started to use PyG.
1. Your demo code develop 3 datasets via maks, while the train&val results is wrong for the val acc is always 1. So I use the index to develop the 3 datasets. Is there any difference?
2. I have tryed some baseline Graph methods in Brain Graph classification while the model is always unstable for the train&val loss and acc are changing in every epoch. Is it appropriate to report and test the even the model is still trainable?

**Comment by cnuzh** [comment]
@Kokakuu I have the same question as you.  Have you solved it now? How can we obtain the results shown in
the article? @xxlya

**Comment by easerbb** [comment]
@cnuzh Hello,  I have the same question as you. Have you solved it now?
